documentation/docs/30-advanced/67-shallow-routing.md

```diff
@@ -22,21 +22,24 @@ SvelteKit makes this possible with the [`pushState`]($app-navigation#pushState)
 	}
 </script>
 
-{#if $page.state.showModal}
+{#if page.state.showModal}
 	<Modal close={() => history.back()} />
 {/if}
 ```
 
-The modal can be dismissed by navigating back (unsetting `$page.state.showModal`) or by interacting with it in a way that causes the `close` callback to run, which will navigate back programmatically.
+The modal can be dismissed by navigating back (unsetting `page.state.showModal`) or by interacting with it in a way that causes the `close` callback to run, which will navigate back programmatically.
 
 ## API
 
 The first argument to `pushState` is the URL, relative to the current URL. To stay on the current URL, use `''`.
 
-The second argument is the new page state, which can be accessed via the [page store]($app-stores#page) as `$page.state`. You can make page state type-safe by declaring an [`App.PageState`](types#PageState) interface (usually in `src/app.d.ts`).
+The second argument is the new page state, which can be accessed via the [page state]($app-state#page) as `page.state`. You can make page state type-safe by declaring an [`App.PageState`](types#PageState) interface (usually in `src/app.d.ts`).
```

Comment:
'via the page state as page.state' is why we can't really call `page` 'page state', we should refer to it as 'the page object' instead

```suggestion
The second argument is the new page state, which can be accessed via the [page object]($app-state#page) as `page.state`. You can make page state type-safe by declaring an [`App.PageState`](types#PageState) interface (usually in `src/app.d.ts`).
```